Makefile

```diff
@@ -34,16 +45,21 @@ lint: $(TOOL_BIN)/golangci-lint
 
 .PHONY: docker
 docker:
-	cd docker && docker buildx build --load --no-cache --platform linux/arm64 -t ghcr.io/viam-modules/raspberry-pi:arm64 .
+	cd docker && docker buildx build --load --no-cache --platform linux/$(DOCKER_ARCH) -t ghcr.io/viam-modules/raspberry-pi:$(DOCKER_ARCH) .
+
+docker-32-bit:
+	cd docker && docker build --platform linux/arm -t ghcr.io/viam-modules/raspberry-pi:arm32 .
 
 .PHONY: docker-upload
 docker-upload:
 	docker push ghcr.io/viam-modules/raspberry-pi:arm64
 
+docker-upload-32:
+	docker push ghcr.io/viam-modules/raspberry-pi:arm32
+
```

Comment:
Same as above build... alias them.

Also, suggest a "docker-all" alias, and "docker-upload" alias that does both architectures to simplify updating everything when needed.